### DISCUSSION TITLE
how to update?

### DISCUSSION THREAD

**Discussion by Domme0506** [zwave-js/zwave-js-ui#3758] [Q&A] [answered]
![image](https://github.com/zwave-js/zwave-js-ui/assets/97409544/adc8c8f7-4d41-440f-ad11-22e051cc5b08)

![image](https://github.com/zwave-js/zwave-js-ui/assets/97409544/d2b2cbe3-3559-4b12-a905-5754be81cc26)

I received this message, but I'm unsure how to proceed with the update.

I have the software installed on a Raspberry Pi within a Docker container, but it is an older version from when it was still named zwavejs2mqtt. Is it possible to update to the new Z-Wave JS UI?

**Comment by robertsLando** [comment, marked as answer]
>  Is it possible to update to the new Z-Wave JS UI?

Sure, just change the name of the container to uuse zwave-js-ui instead of zwavejs2mqtt. If you are using docker-compose use this commands in order to pull the new image locally and start it:

```
docker-compose pull
docker-copose up -d
```

Be sure the volumes are setup correctly in order to don't loose any data as explained in [docs](https://zwave-js.github.io/zwave-js-ui/#/getting-started/docker?id=run-using-volumes)

**Comment by Domme0506** [comment]
thanks a lot. 
it worked like a charm
